new/LIT6366MEDeath.xml

```diff
@@ -0,0 +1,98 @@
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2021-01-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Turaiev1905Eustathii"/>
+                        <citedRange unit="page">167-177 (second half of the 17th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listBibl type="translation">
+                    <bibl>
+                        <ptr target="bm:Turaev1906Ewostatewos"/>
+                        <citedRange unit="page">92-97 (second half of the 17th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6366MEDeath" passive="LIT4315Miracles"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="title" xml:lang="gez">
+                    <note>This title is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ተአምር፡ ዘገብረ፡ አቡነ፡ ኤዎስጣቴዎስ፡ ቅዱስ፡ ከያዴ፡ ባሕር፡ በሐመረ፡ ልብሱ። <gap reason="ellipsis" resp="MV"/>
+                    </ab>
+                </div>
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ስምዑ፡ አሕዛብ፡ ወሕዝብ፡ እለ፡ ሀለውክሙ፡ በምሥራቅ፡ ወበምዕራብ፡ በሴሜን፡ ወበደቡብ፡ በመስዕ፡ ወበአዜ። ወዘገብረ፡ ዘንተ፡ መምሕር፡ አፈ፡ ዘርብ፡ አርዘ፡ 
+                        ሊባኖስ፡ ርጡብ። ውእቱኬ፡ ምጽላለ፡ ብዙኃን፡ አሕዛብ። ወአርጋብ። ምሉአነ፡ ጥበብ፡ ዕፁብኬ፡ ውእቱ፡ ዘከመ፡ አዘዞሙ፡ አቡነ፡ ኤዎስጣቴዎስ፡
+                        ለደቂቁ፡
+                    </ab>
+                </div>
+                <div type="textpart" subtype="explicit" xml:lang="gez">
+                    <note>
+                        This explicit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ወሶበ፡ ሞቱ፡ እሙንቱ፡ አናብስት፡ ቀበርዎሙ፡ ከመ፡ ሰብእ፡ በፀጋመ፡ መቃብሪሁ፡ ለአቡነ፡
+                        ኤዎስጣቴዎስ፡ የአምሩ፡ ከመ፡ ነበሩ፡ ምስሌሁ፡ ፲ወ፬ ዓመታት፡ እንዘ፡ ይትለአክዎ። ወይትዔዘዙ፡ ኀበ፡ ኵሉ፡ ዘሀለየ። ወሶበ፡ ርእየ፡ 
```

Comment:
፲ወ፬ ዓመታት፡
We decided to not place a space after the numeral but this decision is limited to our Mss-entries?

Reply:
I missed this occurrence